src/components/Contacts/ContactsPage.tsx

```diff
@@ -0,0 +1,9 @@
+import { Communication } from "./Communication/Communication";
+
+export default function ContactsPage() {
+  return (
+    <>
+      <Communication />
+    </>
```

Comment:
This you have only one child, no need fragment tags
```suggestion
      <Communication />
```